Twice.dfy(26,21): Error: assertion may not hold
Twice.dfy(34,31): Error: possible violation of function precondition

Dafny program verifier finished with 2 verified, 2 errors
